src/main/java/org/apache/hadoop/mapred/MesosExecutor.java

```diff
@@ -190,6 +189,7 @@ public void revokeSlots() {
     // Be sure there's nothing running and nothing in the launcher queue.
 
     // If we expect to have no slots, let's go ahead and terminate the task launchers
+    // CONDITION IS ALWAYS TRUE
```

Comment:
Feel free to remove both the `if()` statements wrapping this. Partly left around from a previous implementation that is unlikely to be resurrected.